tutorbackup/templates/backup/build/backup/backup_local_services.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+mysqldump --all-databases --password=$MYSQL_ROOT_PASSWORD --user=$MYSQL_ROOT_USERNAME --host=$MYSQL_HOST --port=$MYSQL_PORT > /data/mysql_dump.sql
+mongodump --out=/data/mongodb_dump --host=$MONGODB_HOST --port=$MONGODB_PORT
+cp /caddy/Caddyfile /data/Caddyfile
+tar -zcvf /backup/backup.tar.gz /data/
```

Comment:
Both MySQL and Mongo dumps can get pretty large. Maybe we're better served with `-J` (xz) instead of `-z` (gzip)?

Reply:
makes sense. I'll fix